install/helm/open-match/subcharts/open-match-test/templates/stress-master.yaml

```diff
@@ -26,7 +26,7 @@ spec:
   selector:
     app: {{ template "openmatch.name" . }}
     component: locust-master
-  type: {{ .Values.stresstest.portType }}
+  type: {{ coalesce .Values.global.kubernetes.service.portType .Values.stresstest.portType }}
```

Comment:
so value specified in stresstest will trump the global default (if sepcified?)

Reply:
`coalesce` mean pick the first value that is not empty, so value specified in stresstest will be overrided if we have `.Values.global.kubernetes.service.portType` being set.